### PR TITLE
Bugfix MTE-737 [v112] Update login page for FxA China

### PR DESCRIPTION
**Pull request by clarmso**
The sync integration test `testFxASyncPageUsingChinaFxA()` has been failing because the login page for Firefox Accounts China (https://accounts.firefox.com.cn) has been updated. The existing test assumes the old login page, which is distinct from the regular one. The new Firefox Accounts China login page now has the same look as the regular one's.

Since there's a separate server for the Firefox Accounts China, we would just check if the login page loads instead of attempting to enter the credentials. The temporary credentials generated for other tests are only good for the regular server. 

I've verified that the updated test is passing by the following command:
```
pipenv run pytest test_integration.py::test_sync_china_fxa_server
```

https://mozilla-hub.atlassian.net/browse/MTE-737

### CODE REVIEW

**Comment by isabelrios** [comment]
> The existing test assumes the old login page, which is distinct from the regular one. The new Firefox Accounts China login page now has the same look as the regular one's.

Just to double check, has you confirmed this with Bingqing Li (@bli on slack)? (sorry I can't find her handle here)

File: Tests/XCUITests/IntegrationTests.swift [hunk 1]
```diff
@@ -94,12 +94,12 @@ class IntegrationTests: BaseTestCase {
         navigator.goto(BrowserTabMenu)
         navigator.goto(Intro_FxASignin)
         navigator.performAction(Action.OpenEmailToSignIn)
-        waitForExistence(app.navigationBars[AccessibilityIdentifiers.Settings.FirefoxAccount.fxaNavigationBar], timeout: TIMEOUT_LONG)
-        waitForExistence(app.webViews.textFields[AccessibilityIdentifiers.Settings.FirefoxAccount.emailTextFieldChinaFxA], timeout: TIMEOUT_LONG)
 
-        // Wait for element not present on FxA sign in page China FxA server
-        waitForNoExistence(app.webViews.otherElements.staticTexts["Firefox Monitor"])
-        XCTAssertFalse(app.webViews.otherElements.staticTexts["Firefox Monitor"].exists)
+        sleep(5)
```
Comment (nishant2718) [inline Tests/XCUITests/IntegrationTests.swift:98]: I think 1waitForExistence` should be good enough - I wouldn't sleep the thread unless you really have to. If you do, a quick comment for why would be helpful.

**Comment by clarmso** [comment]
> Just to double check, has you confirmed this with Bingqing Li (@bli on slack)? (sorry I can't find her handle here)

Yes, I confirmed with @bli on March 3. They said that "The sign-in page should be the same with global one".

**Comment by mte-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Edited 1 files</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Created 0 files</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against dcfd80f727b7053ef49414c1b97114b9b0e24cba
</p>

**Review by isabelrios — APPROVED** [review]

**Pull request merged**